configure_product/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+{
+    'name': "Configure Product button in Delivery order & Production order",
+    'category': 'Hidden',
+    'description': """System will add Configure Product button after Add a product button in Delivery order and Production order form. 
```

Comment:
Is this description accurate? I understand it applies to sales orders and purchasing only.